internal/eigenState/avsOperators/avsOperators.go

```diff
@@ -80,12 +78,10 @@ func NewAvsOperatorsModel(
 	globalConfig *config.Config,
 ) (*AvsOperatorsModel, error) {
 	s := &AvsOperatorsModel{
-		BaseEigenState: base.BaseEigenState{
-			Logger: logger,
-		},
-		DB:           grm,
-		logger:       logger,
-		globalConfig: globalConfig,
+		BaseEigenState: base.NewBaseEigenState(logger, grm),
```

Comment:
BaseEigenState is meant to be stateless. The only reason it really exists is to define an interface. If it didnt need to take a logger, then each of it's member functions would be pure, stateless functions. Arguably, that can be done by passing the logger to each function, which is actually more in line with idiomatic Go.